pkg/bisect/bisect.go

```diff
@@ -415,15 +416,15 @@ func (env *env) build() (*vcs.Commit, string, error) {
 		return nil, "", err
 	}
 
-	bisectEnv, err := env.bisecter.EnvForCommit(env.cfg.BinDir, current.Hash, env.kernelConfig)
+	bisectEnv, err := env.bisecter.EnvForCommit(env.cfg.BisectCompiler, env.cfg.BinDir, current.Hash, env.kernelConfig)
 	if err != nil {
-		return nil, "", err
+		return current, "", err
```

Comment:
This can fail for a variety of reasons. I ran into this when bisecting ICB, which subtree merges repos that don't contain any linux source into ICB. During bisection we can end up in those subtrees, which we can't test. Here the error was that no linux version git tags could be found, which we use to choose the correct compiler version. 

If we return nil here, the whole bisection will be aborted. If we return current, current will be skipped and the bisection will progress with another commit.

Reply:
Discussed offline -- for now it's indeed a viable solution, for the future we'd want some much more efficient git subtree bisection.